index.ts

```diff
@@ -94,7 +89,7 @@ export const exportEvents: AvoInspectorPlugin['exportEvents'] = async (events, {
         const trackEventsRes = await fetch('https://api.avo.app/inspector/posthog/v1/track', {
             method: 'POST',
             headers: global.defaultHeaders,
-            body: JSON.stringify(avoEvents),
+            body: JSON.stringify(avoEvent),
```

Comment:
Similar to the others, I just want to check we know this is valid, or should we be sending `[avoEvent]`?

Reply:
better safe than sorry :)